Library/Homebrew/cask/spec/cask/artifact/binary_spec.rb

```diff
@@ -20,7 +20,7 @@
     shutup do
       Hbc::Artifact::Binary.new(cask).install_phase
     end
-    expect(FileHelper.valid_alias?(expected_path)).to be true
+    expect(expected_path).to be_a_valid_symlink
```

Comment:
I'd argue for https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming) but don't feel strongly either way.